src/rmkerr.cpp

```diff
@@ -1026,9 +1027,10 @@ LOCAL int presskey( 	// prompt user after error message display
 #elif !defined( LOGWIN)
 	printf(
 		erAction == ABT
-		? "Press any key (program will abort) "
-		: "Press 'A' to abort, any other key to continue " );
-	int key = getch();							// input any char
+		? "Press any key and enter (program will abort) "
+		: "Press 'A' and enter to abort, any other key to continue " );
+	char key;
+	std::cin >> key;							// input any char
```

Comment:
This is still different behavior but, as I think you've concluded as well, I don't know if there is an alternative. I think the instructions could be a little clearer:

```
? "Enter any character (program will abort) "
: "Enter 'A' to abort, any other character to continue "
```

I'm also curious what happens if the user enters multiple characters.

Reply:
As long as the first character is either `a` or `A` it will take that as an abort. Otherwise, it will continue.